### PR TITLE
Updated DBus plugin

### PR DESCRIPTION
**Pull request by strahlex**
Added new notifications features and updated the README to reflect the changes

### CODE REVIEW

File: src/plugins/dbus/sendmessage.c [hunk 1]
```diff
@@ -1,50 +1,70 @@
...
-	dbus_connection_send (connection, message, NULL);
-	dbus_connection_flush (connection);
-
-	dbus_message_unref (message);
-
-	dbus_connection_unref (connection);
-	dbus_error_free (&error);
-
-	return 1;
+    DBusConnection *connection;
```
Comment (markus2330) [inline src/plugins/dbus/sendmessage.c]: can you avoid converting the tabs to spaces here please?

File: src/plugins/dbus/README.md [hunk 2]
```diff
@@ -59,6 +64,35 @@ then we can receive the notification events using:
 
 	dbus-monitor type='signal',interface='org.libelektra',path='/org/libelektra/configuration'
 
+### Python
+In Python the DBus notifications can be used a s follows
```
Comment (markus2330) [inline src/plugins/dbus/README.md]: Nice description, but here is a type: (as follows:)

**Comment by markus2330** [comment]
Thanks, very nice! Can you quickly fix the spaces, I would like to do the release very soon ;)

**Comment by markus2330** [comment]
add to whitelist

**Comment by markus2330** [comment]
jenkins build please

**Comment by strahlex** [comment]
Hope that fixes it.

**Comment by markus2330** [comment]
(and you can add yourself to AUTHORS)

**Comment by markus2330** [comment]
Perfect! Lets wait for the build server to finish

**Pull request merged**